src/core/helpers/courses.js

```diff
@@ -29,3 +38,25 @@ export const computeDuration = steps => (steps.length
   ? steps.reduce((acc, s) => (s.theoreticalDuration ? acc.add(s.theoreticalDuration) : acc), CompaniDuration()).toISO()
   : PT0S
 );
+
+export const getQuestionnaireTypeTitle = (types) => {
```

Comment:
je ne comprends pas ta logique ; quitte a faire une boucle for, autant aller jusqu'au bout
tu pourrais ecrire ta methode comme suit
```
export const getQuestionnaireTypeTitle = (types) => {
  const labels = [];
  for (const type of types) {
    switch (type) {
      case EXPECTATIONS:
        labels.push('de recueil des attentes');
        break;
      case END_OF_COURSE:
        labels.push('de fin de formation');
        break;
      case SELF_POSITIONNING:
        labels.push('d\'auto-positionnement');
        break;
      default:
        break;
    }
  }

  return labels;
};
```
- puis

Reply:
ensuite il y a un petit suejt pluriel ou singulier sur "au questionnaire" : si tu en a 2 il faut peut etre mettre "aux questionnaires" à voir

Reply:
et ensuite tu peux utiliser `join(', ') sur les labels pour concatener
ou creer une methode qui pour les elements d'une liste, les joint avec des virgules sauf pour le dernier ou tu mets "et"

Reply:
done
perso j'aurai pas mis de "s" vu qu'ils répondent à un seul questionnaire au final, non ?